snap/quota/resources_test.go

```diff
@@ -45,7 +45,20 @@ func (s *resourcesTestSuite) TestQuotaValidation(c *C) {
 	}
 }
 
-func (s *resourcesTestSuite) TestQuotaChangeValidation(c *C) {
+func (s *resourcesTestSuite) TestQuotaValidationPasses(c *C) {
```

Comment:
I guess this is written like this as a list because we expect more of these tests to get added when we add cpu?

Reply:
Exactly, added a comment for now